<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryConditionsWallRadiativeTransferForm</class>
 <widget class="QWidget" name="BoundaryConditionsWallRadiativeTransferForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRadiative">
     <property name="title">
      <string>Thermal radiative transfer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditEmissivity"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelEmissivity">
          <property name="text">
           <string>Emissivity</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Type (gray or black wall)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="comboBoxRadiative">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Code_Saturne keyword: &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;ISOTHP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelConductivity">
          <property name="text">
           <string>Conductivty</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditConductivity"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelConductivityUnit">
          <property name="text">
           <string>W/m/K</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelThickness">
          <property name="text">
           <string>Thickness</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditThickness"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="labelThicknessUnit">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelExtTemperature">
          <property name="text">
           <string>Exterior temperature</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditExtTemperature"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="labelExtTemperatureUnit">
          <property name="text">
           <string>K</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelConductionFlux">
          <property name="text">
           <string>Conduction flux</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditConductionFlux"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="labelConductionFluxUnit">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;W/m&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelIntTemperature">
          <property name="text">
           <string>Interior temperature</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEditIntTemperature"/>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="labelIntTemperatureUnit">
          <property name="text">
           <string>K</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
